cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -149,7 +147,7 @@
 ]
 
 nodejs_versions = %w[
-  7.4.0
+  8.1.3
```

Comment:
this should now be 8.1.4 (which was a security release, so it'd be a good default)